cli/cmd/deploy/deploy.go

```diff
@@ -10,7 +10,7 @@ import (
 
 // DeployCmd is the guided tour for deploying rill projects to rill cloud.
 func DeployCmd(ch *cmdutil.Helper) *cobra.Command {
-	var upload, github bool
+	var upload, github, zipship bool
```

Comment:
Have the variables reflect the non-deprecated flag names:

```suggestion
	var managed, github, archive bool
```